habitat/config/default.py

```diff
@@ -30,6 +30,7 @@
 _C.TASK.SUCCESS_DISTANCE = 0.2
 _C.TASK.SENSORS = []
 _C.TASK.MEASUREMENTS = []
+_C.TASK.GOAL_SENSOR_UUID = "pointgoal"
```

Comment:
Maybe both `_C.TASK.SUCCESS_DISTANCE` and `_C.TASK.GOAL_SENSOR_UUID` worth to move to `_C.TASK.NAV.GOAL_SENSOR_UUID/SUCCESS_DISTANCE` as it doesn't look general for all tasks types.

Reply:
Didn't realize that both `TASK.SUCCESS_DISTANCE` and `TASK.GOAL_SENSOR_UUID` aren't general. Then it looks like there's much more work to be done to generalize agents